pallets/passkey/README.md

```diff
@@ -1,3 +1,33 @@
 # Passkey Pallet
 
 Provides a way to execute transactions using passkey signatures.
+
+## Summary
+
+Passkeys are a secure alternative to passwords for authentication. Due to its ease of use for the
+users and having a well established support in different platforms, we decided to support Passkeys
+in the Frequency chain. With this feature users would be able to execute a transactions by
```

Comment:
nit: Something instead of this
` Due to its ease of use for the
users and having a well established support in different platforms, we decided to support Passkeys
in the Frequency chain.`

To this

` Due to its ease of use for the
users and having a well established support in different platforms, Frequency chain supports passkey p256 signatures for select transactions.`